### DISCUSSION TITLE
create issue using oracle - Validation Failed

### DISCUSSION THREAD

**Discussion by ericyuri** [community/community#24831] [Apps, API and Webhooks]
<p>Hi, </p>
<p>Do you known what I am doing wrong ?</p>
<p>Thank you,</p>
<p>Eric</p>
<pre><code>declare
l_json_clob clob;
l_json_response varchar2(15000);
l_apex_user varchar2(50) := 'ericyuri';
l_apex_pw varchar2(50) := 'my_password';
l_api_create_issue varchar2(100) := 'https://api.github.com/repos/ericyuri/Oracle/issues';
begin
apex_web_service.g_request_headers(1).name := 'User-Agent';
apex_web_service.g_request_headers(1).Value := 'application/json';

l_json_clob := '{
"title": "Found a bug",
"body": "Im having a problem with this.",
"assignees": [
"octocat"
],
"milestone": 1,
"labels": [
"bug"
]
}';
l_json_response := apex_web_service.make_rest_request
( p_url =&gt; l_api_create_issue
, p_http_method =&gt; 'POST'
, p_username =&gt; l_apex_user
, p_password =&gt; l_apex_pw
, p_body =&gt; l_json_clob
, p_wallet_path =&gt; null
);

dbms_output.put_line(l_json_response);

end;
</code></pre>
<p>result</p>
<pre><code>{"message":"Validation Failed","errors":[{"value":1,"resource":"Issue","field":"milestone","code":"invalid"}],"documentation_url":"https://developer.github.com/v3/issues/#create-an-issue"}

Statement processed.
</code></pre>

**Comment by lee-dohm** [comment, marked as answer]
<p>It looks like you’re trying to use <a href="https://developer.github.com/v3/issues/#create-an-issue" rel="nofollow noopener">the example from the documentation</a>. The problem with the example is that if you don’t have any milestones defined in the repository you’re testing with, the creation of the issue will fail. If you take the line about the milestone out, it may work better.</p>
<p>I hope that helps!</p>

**Comment by ericyuri** [comment]
<p>Hi lee-dohmn,</p>
<p>Thank you for your awnser.</p>
<p>I just modified the l_json_clob to </p>
<pre><code>l_json_clob := '{
"title": "Found a bug",
"body": "Im having a problem with this.",
"labels": ["bug"]}';
</code></pre>
<p>and it worked !!</p>
<p>:smiley:</p>
